indexing-service/src/main/java/org/apache/druid/indexing/common/task/CompactionTask.java

```diff
@@ -600,7 +580,7 @@ static Map<Interval, DataSchema> createDataSchemasForIntervals(
     }
   }
 
-  private static List<TimelineObjectHolder<String, DataSegment>> retrieveRelevantTimelineHolders(
+  private static Iterable<DataSegment> retrieveRelevantTimelineHolders(
```

Comment:
I guess the objects being returned is still relevant to the timeline, but does this method need a rename?

Reply:
Fair point, let me include that in the next PR.